sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -95,6 +95,30 @@ var credential = new MetricsAdvisorKeyCredential(subscriptionKey, apiKey);
 var adminClient = new MetricsAdvisorAdministrationClient(new Uri(endpoint), credential);
 ```
 
+#### Create a MetricsAdvisorClient or a MetricsAdvisorAdministrationClient with Azure Active Directory
+
+`MetricsAdvisorKeyCredential` authentication is used in the examples in this getting started guide, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity].
+
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below, or other credential providers provided with the Azure SDK, please install the `Azure.Identity` package:
+
+```PowerShell
+Install-Package Azure.Identity
+```
+
+You will also need to [register a new AAD application][register_aad_app] and [grant access][aad_grant_access] to Metrics Advisor by assigning the `"Cognitive Services Metrics Advisor User"` role to your service principal. You may want to assign the `"Cognitive Services Metrics Advisor Administrator"` role instead if administrator privileges are required.
```

Comment:
I'd love to include a list of what these "administrator privileges" are, but I couldn't find it clearly documented anywhere, and even when testing it myself, the service behavior seems a bit inconsistent. I still need to clarify this topic before adding it to the README. (tracked by https://github.com/Azure/azure-sdk-for-net/issues/18207).